src/components/Breadcrumbs/styles.module.css

```diff
@@ -14,6 +14,20 @@
 }
 
 .locBreadcrumbContainer {
-  padding: 16px 0;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  align-items: center;
+  margin: 16px 0;
   font-size: 12px;
 }
+
+.search {
+  width: 33%;
+}
+
+@media (max-width: 720px) {
+  body .search{
+    width: 100%!important;
```

Comment:
do you need to include `!important` here?

Reply:
I removed `!important` since it is no longer necessary. When I had the `style=width` line it was needed to override the class styling. Thanks for pointing that out